class-cmb2-conditionals.php

```diff
@@ -253,7 +253,7 @@ protected function filter_field_data_to_save( $data_to_save, $field_id, $conditi
 	 * check can be removed once the min version for this plugin has been upped to 4.4.}}
 	 */
 	if ( ( function_exists( 'wp_installing' ) && wp_installing() === false ) || ( ! function_exists( 'wp_installing' ) && ( ! defined( 'WP_INSTALLING' ) || WP_INSTALLING === false ) ) ) {
-		add_action( 'plugins_loaded', 'cmb2_conditionals_init' );
+		add_action( 'after_setup_theme', 'cmb2_conditionals_init' );
```

Comment:
Something like that could work with the class check.  You think all those checks are necessary though? Would there be a time when after_setup_theme would be too late to init that you can think of?